engine/src/main/kotlin/io/provenance/scope/ContractWrapper.kt

```diff
@@ -13,6 +13,7 @@ import io.provenance.scope.objectstore.util.base64Decode
 import io.provenance.scope.util.orThrowContractDefinition
 import io.provenance.scope.util.toOffsetDateTime
 import io.provenance.scope.util.withoutLogging
+import org.bouncycastle.asn1.x500.style.RFC4519Style.name
```

Comment:
What is this import used for in the changes?

Reply:
It's used for stupid IntelliJ auto-importing things 😆  I'll remove it, good eye!